### PR TITLE
Hw1 task02

### PR DESCRIPTION
**Pull request by mirmozavr**
Hw1 task02 with tests

### CODE REVIEW

File: hw1/task02/test_task02.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    [
+        (true_fib_seq, True),
+        ([0], True),
+        ([0, 1], True),
+        ([0, 0], False),
+        ([0, -1], False),
+        ([0, 1, 1, 2, 3, 5, 8], True),
+        ([], False),
+        ([-5, 0, 1, 1, 2], False),
+        ([1, 1, 2], False),
```
Comment (Cjkjvfnby) [inline hw1/task02/test_task02.py:40]: Only one case for 3 digits?  I expect at least 2 positive and negative.

File: hw1/task02/task02.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+"""
+Given a cell with "it's a fib sequence" from slideshow.
+
+please write function "check_fib", which accepts a Sequence of integers, and
+returns if the given sequence is a Fibonacci sequence
+We guarantee, that the given sequence contain >= 0 integers inside.
+"""
+
+
+def check_fib(data: list) -> bool:
```
Comment (Cjkjvfnby) [inline hw1/task02/task02.py:10]: Let try to write it in a different way. Too many ifs make it hard to read.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: hw1/task02/test_task02.py (hunk 1 above)
Comment (mirmozavr) [reply hw1/task02/test_task02.py:40]: Agreed

File: hw1/task02/task02.py (hunk 2 above)
Comment (mirmozavr) [reply hw1/task02/task02.py:10]: Will rewrite it with len and in check

File: hw1/task02/task02.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    # process short data sequences (2 or less)
+    if len(data) < 3:
+        if data in ([0], [0, 1]):
+            return True
+        else:
+            return False
```
Comment (Cjkjvfnby) [inline hw1/task02/task02.py]: You can simplify that lines even more.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: hw1/task02/task02.py (hunk 3 above)
Comment (mirmozavr) [reply hw1/task02/task02.py]: Absolutely.
return data in ...........

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**